src/components/verification-grid-tile/verification-grid-tile.ts

```diff
@@ -291,26 +294,57 @@ export class VerificationGridTileComponent extends SignalWatcher(AbstractCompone
     `;
   }
 
-  private meterSegmentsTemplate(): TemplateResult<1> {
+  private classificationMeterTemplate(): TemplateResult {
     return html`
       ${repeat(this.requiredTags, (tag: Tag) => {
-        const decision = this.model.decisions.get(tag.text);
+        const decision = this.model.classifications.get(tag.text);
         const decisionText = decision ? decision.confirmed : "no decision";
 
         let color: string | undefined;
         if (decision && decision.confirmed !== DecisionOptions.SKIP) {
           color = this.injector.colorService(decision);
         }
 
-        return html`
-          <sl-tooltip content="${tag.text ?? tag} (${decisionText})">
-            <span class="progress-meter-segment" style="background-color: var(${ifDefined(color)})"></span>
-          </sl-tooltip>
-        `;
+        return this.meterSegmentTemplate(`${tag.text} (${decisionText})`, color);
       })}
     `;
   }
 
+  private verificationMeterTemplate(): TemplateResult | typeof nothing {
+    const currentVerificationModel = this.model.verification;
+    const decisionText = currentVerificationModel ? currentVerificationModel.confirmed : "no decision";
+    const tooltipText = `verification: ${this.model.tag.text} (${decisionText})`;
+
+    // if there is no verification decision on the tiles subject model, then
+    // return the verification meter segment with no color
+    if (!currentVerificationModel) {
+      return this.meterSegmentTemplate(tooltipText);
+    }
+
+    const meterColor = this.injector.colorService(currentVerificationModel);
+    return this.meterSegmentTemplate(tooltipText, meterColor);
+  }
+
+  private meterSegmentTemplate(tooltip: string, color?: string): TemplateResult {
+    return html`
+      <sl-tooltip content="${tooltip}">
+        <span class="progress-meter-segment" style="background-color: var(${ifDefined(color)})"></span>
+      </sl-tooltip>
+    `;
+  }
+
+  private progressMeterTemplate(): TemplateResult {
+    // prettier wants to format this as a single line because it thinks it is
+    // string interpolation
+    // to improve readability, I have disabled prettier for this line so that we
+    // can put each of the templates on a separate line
+    // prettier-ignore
+    return html`
+      ${this.classificationMeterTemplate()}
+      ${when(this.requiresVerification, () => this.verificationMeterTemplate())}
```

Comment:
does this change the order of meter segments? it looks like verification always appears last? Shouldn't order be kept in the same order as the decisions are templated in?